redhat-ipaas-dev-single-tenant.yml

```diff
@@ -365,6 +370,8 @@ objects:
             value: "true"
           - name: JAVA_OPTIONS
             value: "-Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStore=/tls-keystore/openshift-truststore.jks -Duser.home=/tmp"
+            - name: ENDPOINTS_TEST_SUPPORT_ENABLED
```

Comment:
I see. Maybe we should even move this specific endpoint into an extra maven module even.

Actually, I'm still not so happy to have this endpoint in the application which could potentially switched on by a deployment option. Since all tests should go on the final thing (where you actually need this endpoint), its probably better to have the fixture setup _external_ to the app (and calling directly to the DB underneath). 

Its not something for now but maybe we should keep that in mind.